src/frontends/tensorflow_common/src/op/segment_sum.cpp

```diff
@@ -46,8 +50,18 @@ OutputVector translate_segment_sum_op(const NodeContext& node) {
     auto indices = make_shared<v4::Range>(const_zero, num_indices, const_one, indices_type);
 
     auto emb_segment_sum = make_shared<v3::EmbeddingSegmentsSum>(data, indices, segment_ids, num_segments);
```

Comment:
here you pass `ComplexTypeMark` output for complex case but you need to pass floating-point representation of complex tensor: `complex_type_mark ->input(0)`. But it is needed only for complex case.

Reply:
Done 
![image](https://github.com/openvinotoolkit/openvino/assets/116452077/cc67ef63-2b96-479c-85dd-bf182a5d39f4)


This Should Work